sycl/include/CL/sycl/detail/context_impl.hpp

```diff
@@ -121,6 +121,23 @@ class context_impl {
   ///
   /// @return a pointer to USM dispatcher.
   std::shared_ptr<usm::USMDispatcher> getUSMDispatch() const;
+
+  /// In contrast to user programs, which are compiled from user code, library
+  /// programs come from the SYCL runtime. They are identified by the
+  /// corresponding extension name:
+  ///
+  ///  "cl_intel_devicelib_assert" -> #<pi_program with assert functions>
```

Comment:
Can we have enum values instead of strings?

Reply:
Done.